### DISCUSSION TITLE
Which magick command is used on this image (example)?

### DISCUSSION THREAD

**Discussion by Venus642** [ImageMagick/ImageMagick#4717] [General] [answered]
Hello.
I have a general question about this following image (example): 
[delete]
With -resize 600x300 I got this photo: 
[delete]
How do you get the image via magick cmd batch command -resize 600x300 (was generated with unknown command)?
[delete]
Thanks for your help.

**Comment by snibgo** [comment, marked as answer]
> With -resize 600x300 I got this photo:

Please show your complete command. "-resize" does not crop images. Both of your outputs have been cropped.

A command that is close to your required output is:
```
magick 1496kitten.jpg -resize "600x300^" -gravity south -crop 600x300+0+0 +repage out.png
```

**Reply by Venus642** [reply]
Big thanks snibgo. This is it!
